src/plugins/controls/common/options_list/suggestions_sorting.ts

```diff
@@ -10,13 +10,17 @@ import { Direction } from '@elastic/eui';
 
 export type OptionsListSortBy = '_count' | '_key';
 
-export const DEFAULT_SORT: SortingType = { by: '_count', direction: 'desc' };
+export const OPTIONS_LIST_DEFAULT_SORT: OptionsListSortingType = {
+  by: '_count',
+  direction: 'desc',
+};
+
+export const optionsListSortDirections: Readonly<Direction[]> = ['asc', 'desc'] as const;
```

Comment:
Related to [the comment above](https://github.com/elastic/kibana/pull/146241/files#r1052707985), I realized I missed some types when I was renaming 🤦  So I took care of that here, as well.

Reply:
I'm wondering why you go through the trouble of creating a new type out of the EUI direction type like this when you could use the EUI type directly or just do a simple union like:

``` typescript
type OptionsListSortDirection = 'asc' | 'desc';
```

I see what you've done here, and how it works, but I'm not sure I see the benefit?